examples/tokenvm/avalanche-ops.aws.md

```diff
@@ -0,0 +1,249 @@
+
+### Step 4
+
+Now that the network and nodes are up, let's install two subnets, each of which runs its own TokenVM. Note that the above, successful `avalancheup-aws apply` run will output the following commands as an example, which you can customize depending on the VMs:
+
+```bash
+# EXAMPLE: write subnet config
+avalancheup-aws subnet-config \
+--log-level=info \
+--proposer-min-block-delay 250000000 \
+--file-path /tmp/subnet-config.json
+
+# EXAMPLE: install subnet + custom blockchain in all nodes
+avalancheup-aws install-subnet-chain \
+--log-level info \
+--region us-west-2 \
+--s3-bucket avalanche-ops-***-us-west-2 \
+--s3-key-prefix aops-custom-***/install-subnet-chain \
+--ssm-doc aops-custom-***-ssm-install-subnet-chain \
+--chain-rpc-url http://52.88.8.107:9650 \
+...
+```
+
+For instance, use the following to **deploy two TokenVM subnets and create separate TokenVM chains** (similar to [scripts/run.sh](./scripts/run.sh)):
+
+```bash
+avalancheup-aws subnet-config \
+--log-level=info \
+--proposer-min-block-delay 0 \
+--file-path /tmp/subnet-config.json
+
+cat <<EOF > /tmp/allocations.json
+[{"address":"token1rvzhmceq997zntgvravfagsks6w0ryud3rylh4cdvayry0dl97nsjzf3yp", "balance":1000000000000}]
+EOF
+rm -f /tmp/tokenvm-genesis.json
+/tmp/token-cli genesis generate /tmp/allocations.json \
+--genesis-file /tmp/tokenvm-genesis.json
+cat /tmp/tokenvm-genesis.json
+
+cat <<EOF > /tmp/tokenvm-chain-config.json
+{
+  "mempoolSize": 10000000,
+  "mempoolExemptPayers":["token1rvzhmceq997zntgvravfagsks6w0ryud3rylh4cdvayry0dl97nsjzf3yp"],
+  "parallelism": 5,
+  "trackedPairs":["*"],
+  "logLevel": "info",
+  "stateSyncServerDelay": 0
+}
+EOF
+cat /tmp/tokenvm-chain-config.json
+```
+
+Note that `--chain-name tokenvm*` and `--node-ids-to-instance-ids` values are different to split nodes into two separate subnets. And `avalancheup-aws install-subnet-chain` **will add the specified nodes as primary network validators (regardless of it's for customer or public networks) before adding them as subnet validators**. So, please make sure the `--key` has enough balance and use `--primary-network-validate-period-in-days` and `--subnet-validate-period-in-days` flags to set custom validate priods (defaults to 16-day for primary network, 14-day for subnet validation):
+
+```bash
+# this will prompt you to confirm the outcome!
+# so make sure to check the outputs!
+avalancheup-aws install-subnet-chain \
+--log-level info \
+--region us-west-2 \
+--s3-bucket avalanche-ops-***-us-west-2 \
+--s3-key-prefix aops-custom-***/install-subnet-chain \
+--ssm-doc aops-custom-***-ssm-install-subnet-chain \
+--chain-rpc-url http://52.88.8.107:9650 \
+--key 0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027 \
+--primary-network-validate-period-in-days 16 \
+--subnet-validate-period-in-days 14 \
+--subnet-config-local-path /tmp/subnet-config.json \
+--subnet-config-remote-dir /data/avalanche-configs/subnets \
+--vm-binary-local-path /tmp/tokenvm \
```

Comment:
We should add instructions for generating a linux-compatible build

Reply:
Ran into this on my mac for example:
```
Building tokenvm in ./build/tHBYNu8ikqo4MWMHehC9iKB9mR5tB3DWzbkYmTfe9buWQ5GZ8
# runtime/cgo
linux_syscall.c:67:13: error: implicit declaration of function 'setresgid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
linux_syscall.c:67:13: note: did you mean 'setregid'?
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/unistd.h:593:6: note: 'setregid' declared here
linux_syscall.c:73:13: error: implicit declaration of function 'setresuid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
linux_syscall.c:73:13: note: did you mean 'setreuid'?
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/unistd.h:595:6: note: 'setreuid' declared here
```

Reply:
Using this to try and unblock: https://words.filippo.io/easy-windows-and-linux-cross-compilers-for-macos/

```
brew install FiloSottile/musl-cross/musl-cross
```

Reply:
Attempting to pre-build to make it easier: https://github.com/ava-labs/hypersdk/pull/112

Reply:
I merged this change...so we can just say to download from github?